cogs/onmyoji.py

```diff
@@ -1,18 +1,110 @@
+from __future__ import print_function
 import discord
 import json
 from discord.ext import commands
+import os.path
+from googleapiclient.discovery import build
+from googleapiclient.http import *
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from google.oauth2 import service_account
+import io
+import csv
+import config
+import re
 
+permission = 'You do not have permission to use this command.'
```

Comment:
So we should move these to a constants.py file (or the config). Since we probably want this to be the same msg across the board.

Also `permission_msg` might be a better var name

Reply:
I have different permission msg's depending on the check but you're not wrong, I'll fix that up.